.github/workflows/ci.yml

```diff
@@ -595,12 +595,20 @@ jobs:
         id: combinecoverage
         run: |
           python -m coverage combine
-          python -m coverage report --fail-under=100
+          echo "## Python Coverage" >> $GITHUB_STEP_SUMMARY
+          echo '```' >> $GITHUB_STEP_SUMMARY
+          python -m coverage report -m --fail-under=100 >> $GITHUB_STEP_SUMMARY
+          echo '```' >> $GITHUB_STEP_SUMMARY
       - name: Combine rust coverage and fail if it's <100%.
         id: combinerustcoverage
         run: |
           sudo apt-get install -y lcov
-          lcov $(for f in *.lcov; do echo --add-tracefile "$f"; done) -o combined.lcov | grep lines | grep "100.0%"
+          RUST_COVERAGE_OUTPUT=$(lcov $(for f in *.lcov; do echo --add-tracefile "$f"; done) -o combined.lcov | grep lines)
```

Comment:
I think we should put the `grep lines` down on 611 so that the summary contains more info?